services/bettersocial/post/blockAnonymousPost.js

```diff
@@ -68,6 +67,7 @@ const BetterSocialBlockAnonymousPost = async (token, selfUserId, postId, source,
         {transaction: t, message, postId, reason, isAnonymous: true}
       );
     });
+    await Getstream.feed.unfollowMainFeedFollowingAnon(token, selfUserId, authorAnonymousUserId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new line has been added to unfollow the main feed following an anonymous user. Ensure that the `unfollowMainFeedFollowingAnon` function handles any potential exceptions, as unhandled exceptions here could cause the program to crash or behave unexpectedly.

```diff
+    try {
+      await Getstream.feed.unfollowMainFeedFollowingAnon(token, selfUserId, authorAnonymousUserId);
+    } catch (e) {
+      console.error('Error in unfollowing main feed:', e.message);
+    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->